frontend/src/bundles/workouts/pages/workout-item/workout-items.tsx

```diff
@@ -34,7 +34,7 @@ const WorkoutItem = (): JSX.Element => {
             <WorkoutStats workout={currentWorkout} />
             {currentWorkout.provider === OAuthProvider.STRAVA && (
                 <Link
-                    to={AppRoute.STRAVA}
+                    to={AppRoute.STRAVA_ATHLETE_TRAINING}
```

Comment:
We shouldn't define this route in as our app route, create a constant for it somewhere.
 Also, I think we should skip client-side routing for this link. Using the `reloadDocument` prop in NavLink should be enough.